src/ray/raylet/raylet.cc

```diff
@@ -108,6 +107,23 @@ void Raylet::Stop() {
   acceptor_.close();
 }
 
+void Raylet::TryToGetAgentInfoAndRegisterGcs() {
+  rpc::AgentInfo agent_info;
+  auto status = node_manager_.TryToGetAgentInfo(&agent_info);
+  if (status.ok()) {
+    self_node_info_.mutable_agent_info()->CopyFrom(agent_info);
+    RAY_CHECK_OK(RegisterGcs());
+  } else {
+    // Because current function and `AgentManager::HandleRegisterAgent`
+    // will be invoke in same thread, so we need post current function
+    // into main_service_ after interval milliseconds.
+    std::this_thread::sleep_for(std::chrono::milliseconds(
+        RayConfig::instance().raylet_get_agent_info_interval_ms()));
```

Comment:
Does raylet_get_agent_info_interval_ms have to be a config? When do we need to adjust this value?

Reply:
I just think that we should avoid hard-code here, so as to avoid having to adjust the number by modifying the code later.